tests/Unit/bootstrap.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+$rootDirectory = getenv('_PS_ROOT_DIR_') ?: __DIR__ . '/../../../../';
+$projectDir = __DIR__ . '/../../';
+require_once $rootDirectory . 'config/config.inc.php';
+require_once $rootDirectory . 'vendor/autoload.php';
+require_once $projectDir . 'vendor/autoload.php';
```

Comment:
Goal is to avoid require PrestaShop classes to launch Unit Tests, using mock of Adapter
Wish we can remove this

Reply:
But if we don't load PrestaShop classes, then some classes like Order, Cart won't work. Unless we load these objects from our created repository which in tests could return a simple object

Reply:
We should avoid that and encapsulate Core classes interaction in some services like DataProvider or Handlers
We will do it step by step I think but it's the final goal